### PR TITLE
Updated README in all sample apps

### PR DESCRIPTION
**Pull request by rominirani**
Updated the README file in all sample applications to include an informational message at the top that before the various commands are run, they will need to be in the correct folder. 

Often folks forget to be in the right application folder before executing the commands. This causes the commands to fail. The addition of the instruction above will help to ensure that the user is in the right folder and then the commands are run.

### CODE REVIEW

File: language/sample-apps/chat-flask-cloudrun/README.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ This application demonstrates a Cloud Run application that has a simple Form-bas
 
 ## Build and Deploy the application to Cloud Run
 
+> NOTE: **Before you move forward, ensure that you have followed the instructions in [SETUP.md](../SETUP.md).**
+Additionally, ensure that you have cloned this repository and are currently in the ```chat-flask-cloudrun``` folder for the rest of the commands.
```
Comment (polong-lin) [inline language/sample-apps/chat-flask-cloudrun/README.md:10]: In terms of phrasing -- could you clarify exactly what you mean by "currently in the ____ folder" so users know what to do to make sure they are currently in the folder? For example, "currently in the ____ folder and it is your active working directory for the rest of the commands" or something like that?

Comment (rominirani) [reply language/sample-apps/chat-flask-cloudrun/README.md]: Thanks. I have rephrased it now across all the READMEs in this PR.

**Comment by polong-lin** [comment]
@rominirani Thanks! Sorry for the trouble, but the Lint Code Base is complaining about Markdown styling -- could you fix the errors before I merge?

Here are the errors:
https://github.com/GoogleCloudPlatform/generative-ai/actions/runs/6875637726/job/18699627765?pr=255

**Comment by rominirani** [comment]
@polong-lin -- I have fixed the Linting errors.

**Review by polong-lin — APPROVED** [review]

**Pull request merged**